requests/compat.py

```diff
@@ -60,6 +61,7 @@
     from http.cookies import Morsel
     from io import StringIO
     from collections import OrderedDict
+    from collections.abc import Callable, Mapping, MutableMapping
```

Comment:
@rhettinger on which versions of Python 3 will this actually succeed on? It was my impression that `collections.abc` was introduced in 3.6 or 3.7.

Reply:
@sigmavirus24 I'd tested it back to 3.4 without issue, so I think this should work.

Reply:
This works back to Python 3.3.